Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -631,13 +634,9 @@ - (void)writeObjectToDefaults:(NSString *)key value:(NSObject *)value {
 
 - (void)persistPrefsToDisk {
     @synchronized (self) {
-        NSDictionary *persistenceDict = [self.persistenceDict copy];
+        if (!self.persistenceDict) return;
```

Comment:
You can click this `copy]` line in Github while reviewing the master branch code in the browser and then scroll up and press the 'Blame' button. It'll show you exactly who committed this with hopefully some explanation of why.